src/Protocol/OStatus.php

```diff
@@ -2201,7 +2209,8 @@ public static function feed($owner_nick, &$last_update, $max_items = 300, $filte
 			if (Config::get('system', 'ostatus_debug')) {
 				$item['body'] .= '🍼';
```

Comment:
😄

Reply:
Look for the other occurrence where it add some character for salmon ...